misk/src/main/kotlin/misk/hibernate/SchemaMigrator.kt

```diff
@@ -37,6 +37,9 @@ internal class SchemaMigrator(
 ) {
   /** Returns a map from version to path. */
   fun availableMigrations(): NavigableMap<Int, String> {
+    if (config.migrations_path == null) {
```

Comment:
hmmmm . . . is this something to support?

Reply:
vttestserver (the Vitess test cluster) supports migrations out of the box. You have to restart it to re-apply new migrations but it's really pretty damn fast to restart (seconds). We could potentially restart it before each test...